auth-web/src/components/pay/UnlinkedShortNameTable.vue

```diff
@@ -0,0 +1,448 @@
+      col,
+      customFilter: {
+        filterApiFn: hasFilter ? (val: any) => loadTableData(col, val || '') : null,
+        clearable: true,
+        label,
+        type,
+        value
+      },
+      hasFilter,
+      minWidth,
+      value
+    })
+
+    const headers = [
+      createHeader('shortName', 'Bank Short Name', 'text', 'Bank Short Name'),
+      // createHeader('depositDate', 'Initial Payment Received Date', 'text', ''),
+      {
+        col: 'depositDate',
+        hasFilter: false,
+        label: 'Initial Payment Received Date',
+        itemFn: (val: any) => {
+          // Example format: 2023-03-11T00:55:05.909229 without timezone
+          const createdOn = moment.utc().toDate()
+          return CommonUtils.formatDisplayDate(createdOn, 'MMMM DD, YYYY<br/>h:mm A')
+        },
+        value: 'Initial Payment Received Date',
+        minWidth: '165px'
+      },
+      createHeader('depositAmount', 'Initial Payment Amount', 'text', 'Initial Payment Amount'),
+      {
+        col: 'actions',
+        hasFilter: false,
+        value: 'Actions',
+        minWidth: '200px'
+      }
+    ]
+
+    const state = reactive({
+      results: [
+        {
+          accountName: 'RCPV',
+          shortName: 'RCPV',
+          accountBranch: 'Saanich',
+          accountId: '3199',
+          id: 1
+        }
+      ],
+      totalResults: 1,
+      filters: {
+        isActive: false,
+        pageNumber: 1,
+        pageLimit: 20,
+        filterPayload: {
+          shortName: '',
+          depositDate: '',
+          depositAmount: ''
+        }
+      } as UnlinkedShortNameFilterParams,
+      loading: false
+    })
+
+    const title = computed(() => {
+      return `Unlinked Bank Short Names (${state.totalResults})`
+    })
+
+    onMounted(async () => {
+      headers.forEach((header) => {
+        if (header.hasFilter) {
+          (header.customFilter as any).filterApiFn = (val: any) => loadTableData(header.col, val || '')
+        }
+      })
+      await loadTableData()
+    })
+
+    function formatAmount (amount: number) {
+      return CommonUtils.formatAmount(amount)
+    }
+
+    function formatDate (date: string) {
+      return CommonUtils.formatCurrentDate(date)
+    }
+
+    async function clickDatePicker () {
+      showDatePicker.value = true
+      // await for datePicker ref to update
+      await nextTick()
+      // datePicker.value.$el.scrollIntoView({ behavior: 'smooth', block: 'center' });
+    }
+
+    function updateDateRange ({ endDate, startDate }: { endDate?: string, startDate?: string }): void {
+      showDatePicker.value = false
+      dateRangeSelected.value = !!(endDate && startDate)
+      if (!dateRangeSelected.value) { endDate = ''; startDate = '' }
+      // loadTableData('depositDate', { endDate, startDate })
+      loadTableData('depositDate', endDate)
+    }
+
+    function handleFilters (filterField?: string, value?: any): void {
```

Comment:
need to refactor handleFilters, loadTableData.. probably do they take state as parameter? and put it in composable